assets/js/components/Dashboard/PendingScreensPage.tsx

```diff
@@ -154,6 +156,19 @@ const PendingScreensPage: ComponentType = () => {
             }
           )}
         </Accordion>
+      </>
+    );
+  } else {
+    layout = (
+      <div className="no-screens-text h5">There are no pending screens.</div>
```

Comment:
I'm not familiar with the design system here, curious why `<div class="h5">` and not `<h5>`?

Reply:
I want to say the original reason was due to how Mary defined the `h5` style, and that style didn't match up with the properties of an `h5` element. In hindsight, we probably should have pushed back a bit and suggested better naming. Typography can be found [here](https://www.figma.com/file/FKWRVrSdoMjZOLorhuykvc/Permanent-Configuration?type=design&node-id=915-31883&mode=design&t=FnHKPCmAspNX09TL-4).